packages/fxa-auth-server/fxa-oauth-server/lib/db/memory.js

```diff
@@ -309,6 +309,63 @@ MemoryStore.prototype = {
     return helpers.aggregateActiveClients(activeClientTokens);
   },
 
+  /**
+   * Get all access tokens for a given user.
+   * @param {String} uid User ID as hex
+   * @returns {Promise}
+   */
+  getAccessTokensByUid: async function getAccessTokensByUid(uid) {
+    if (! uid) {
+      return P.reject(new Error('Uid is required'));
+    }
+    const accessTokens = [];
+    Object.keys(this.tokens).forEach(id => {
```

Comment:
Why not `Object.entries` since you want the token as well?

Reply:
Huh, TIL about `Object.entries` and `Object.values. Thanks!